### ISSUE TITLE
Request: Notify of DNS issues

### ISSUE THREAD

**Issue by parawanderer** [bitcoin-wallet/bitcoin-wallet#861]
I've had this app for a while on my android phone and recently decided to start using a DNS-level ad blocker. 

I decided to send some BTC to a wallet on my phone today and couldn't figure out why the app was not showing the finalised transaction for the wallet that was clearly completed on the blockchain. I observed just the "0 peers connected" message, but this really didn't make me think of "DNS issue". 

I went looking for this "0 peers connected" message for the app and saw some random, unrelated discussion regarding DNS and thought "oh yeah, it could be DNS", which of course it ended up being (at least I'm fairly sure these days non-CS background people interact with BTC without knowing what DNS is). 
I'm not sure that somebody without a background knowledge on DNS would be able to figure something like this out. 

I wanted to suggest/request some sort of error/info message pop up regarding multiple failed DNS resolving attempts over a prolonged period of time & suggestions on approaches to fix the issue.

**Comment by schildbach** [comment]
Yes, some notification for connection problems would make sense.

You can send the report to yourself by replacing the "to" email address in your email app when it opens. Currently there is no way to directly look at the logs in the app, but honestly I think the typical screen is too small for that purpose anyway.

**Comment by 11ANAS** [comment]
Hi
